tecken/base/admin.py

```diff
@@ -0,0 +1,75 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I cribbed this from Socorro. It lets us see the audit log created by any changes that happen in the Django admin.